runtime/permissions/prompter.rs

```diff
@@ -354,6 +364,20 @@ impl PermissionPrompter for TtyPrompter {
         )
         .unwrap();
       }
+      if let Some(stack) = stack {
+        let len = stack.len();
+        for (idx, frame) in stack.into_iter().enumerate() {
+          writeln!(
+            &mut output,
+            "┃  {} {}",
+            colors::gray(if idx != len - 1 { "├─" } else { "└─" }),
+            colors::gray(deno_core::error::format_frame::<
+              deno_core::error::NoAnsiColors,
+            >(&frame))
+          )
+          .unwrap();
+        }
```

Comment:
We need to account for the no of lines added here further down when we clear the terminal